schema_utils.py

```diff
@@ -55,8 +55,13 @@
 def normalize_against_schema(obj, schema, apply_custom_validators=True):
     """Validate the given object using the schema, normalizing if necessary.
 
+    Args:
+        obj: *. The object to validate and normalize.
+        schema: dict. The schema to validate and normalize the value against.
+        apply_custom_validators: bool. Validate the normalized object.
```

Comment:
bool. Whether to validate the normalized object using the validators defined in the schema.

Reply:
Hi @DiegoCaraballo, I mean by my previous comment that please reply to all these open review comments simply by replying 'Done' and I think they have been addressed. Thanks!